NitroxClient/GameLogic/ChatUI/PlayerChatManager.cs

```diff
@@ -78,14 +89,32 @@ public void SendMessage()
 
         private IEnumerator LoadChatLogAsset()
         {
-            yield return AssetBundleLoader.LoadUIAsset("chatlog", "PlayerChatCanvas", true, playerChatGameObject =>
+            yield return AssetBundleLoader.LoadUIAsset(CHAT_LOG_ASSET, "PlayerChatCanvas", true, playerChatGameObject =>
             {
                 playerChat = playerChatGameObject.AddComponent<PlayerChat>();
             });
 
             yield return playerChat.SetupChatComponents();
         }
 
-        public static void LoadChatKeyHint() => Player.main.StartCoroutine(AssetBundleLoader.LoadUIAsset("chatkeyhint", "ChatKeyCanvas"));
+        public void LoadChatKeyHint()
+        {
+            if (!PlayerPrefs.HasKey("Nitrox.chatUsed"))
+            {
+                Player.main.StartCoroutine(AssetBundleLoader.LoadUIAsset(CHAT_KEY_HINT_ASSET, "ChatKeyCanvas", false, chatKeyHintGameObject =>
+                {
+                    chatKeyHint = chatKeyHintGameObject;
+                }));
+            }
+        }
+
+        //TODO: Has to be reworked if the config API for NitroxClient is finished.
+        private void DisableChatKeyHint()
+        {
+            chatKeyHint.GetComponentInChildren<Text>().CrossFadeAlpha(0, 1, false);
+            chatKeyHint.GetComponentInChildren<Image>().CrossFadeAlpha(0, 1, false);
+            PlayerPrefs.SetInt("Nitrox.chatUsed", 1);
```

Comment:
Would be good to have this in a property on this class. So you can just set/get it.

Reply:
Do you mean that the string `Nitrox.chatUsed` should be defined as a const?

Reply:
No, like this:
```cs
private bool ChatUsed
{
    get => PlayerPrefs.GetInt("Nitrox.chatUsed") == 1;
    set => PlayerPrefs.SetInt("Nitrox.chatUsed", value ? 1 : 0);
}
```